### PR TITLE
Modify config for modulization

### PR DESCRIPTION
**Pull request by ArcturusZhang**
- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] The PR targets the `latest` branch.
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] Apache v2 license headers are included in each file.
 
[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/master/CHANGELOG.md

### CODE REVIEW

File: moduler-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+pool:
```
Comment (jhendrixMSFT) [inline moduler-pipelines.yml]: How does this pipeline get invoked?

File: swagger_to_sdk_config.json [hunk 2]
```diff
@@ -1,10 +1,9 @@
 {
   "$schema": "https://openapistorageprod.blob.core.windows.net/sdkautomation/prod/schemas/swagger_to_sdk_config.schema.json",
   "meta": {
-    "after_scripts": [
+    "after_scripts_in_repo": [
```
Comment (jhendrixMSFT) [inline swagger_to_sdk_config.json]: How does `after_scripts_in_repo` change things?

File: swagger_to_sdk_config.json [hunk 3]
```diff
@@ -13,8 +12,10 @@
       "verbose": "",
       "sdkrel:go-sdk-folder": ".",
       "multiapi": "",
-      "use-onever": "",
-      "preview-chk": ""
+      "preview-chk": "",
+      "go.clear-output-folder": false,
+      "stage": "",
```
Comment (jhendrixMSFT) [inline swagger_to_sdk_config.json]: So we're ready to turn this on then?

File: moduler-pipelines.yml (hunk 1 above)
Comment (ArcturusZhang) [reply moduler-pipelines.yml]: This is a new CI, which only triggers when master branch gets new commits.
We have a testing repo [here](https://github.com/test-repo-arcturus/azure-sdk-for-go) which already gets this pipeline enabled

File: swagger_to_sdk_config.json (hunk 2 above)
Comment (ArcturusZhang) [reply swagger_to_sdk_config.json]: This is per what Phoenix recommends, to emphasize this script runs in the root dir and only run once, even there are multiple changed services

**Pull request closed without merging**